biz.aQute.repository/src/aQute/bnd/repository/maven/pom/provider/BndPomRepository.java

```diff
@@ -88,13 +109,118 @@ public synchronized void init() {
 				repository.close();
 				throw new IllegalStateException("We have neither a pom, revision, or query set!");
 			}
+
 			bridge = new BridgeRepository(repoImpl);
 			inited = true;
 		} catch (Exception e) {
 			throw Exceptions.duck(e);
 		}
 	}
 
+	private void startPoll() {
+		Workspace ws = registry.getPlugin(Workspace.class);
+		if ((ws != null) && (ws.getGestalt().containsKey(Constants.GESTALT_BATCH)
+				|| ws.getGestalt().containsKey(Constants.GESTALT_CI)
+				|| ws.getGestalt().containsKey(Constants.GESTALT_OFFLINE))) {
+			return;
+		}
+		final AtomicBoolean busy = new AtomicBoolean();
+		int polltime = configuration.pollTime(DEFAULT_POLL_TIME);
+		if (polltime > 0) {
+			pomPoller = Processor.getScheduledExecutor().scheduleAtFixedRate(new Runnable() {
+
+				@Override
+				public void run() {
+					if (busy.getAndSet(true))
+						return;
+					try {
+						boolean needsUpdate = false;
+						for (Entry<URI,Long> pomFilesEntry : pomFiles.entrySet()) {
+							final URI pomURI = pomFilesEntry.getKey();
+							if ("file".equalsIgnoreCase(pomURI.getScheme())) {
+								if (pomFilesEntry.getValue() == null) {
+									// not interested
+									continue;
+								}
+								File f = new File(pomURI);
+								if (f.isFile() && f.lastModified() > pomFilesEntry.getValue()) {
+									needsUpdate = true;
+									pomFilesEntry.setValue(f.lastModified());
+								}
+							} else {
+								// check for staleness of non-file-URI
+								if (isStale(pomURI))
+									needsUpdate = true;
+							}
+						}
+						if (needsUpdate) {
+							refresh();
+							for (RepositoryListenerPlugin listener : registry
+									.getPlugins(RepositoryListenerPlugin.class))
+								listener.repositoryRefreshed(BndPomRepository.this);
+						}
+
+					} catch (Exception e) {
+						reporter.exception(e, "Error when polling for %s for change", this);
+					} finally {
+						busy.set(false);
+					}
+				}
+
+			}, polltime, polltime, TimeUnit.MILLISECONDS);
```

Comment:
This should be `TimeUnit.SECONDS` since that is what we use for other repos. It is not useful to configure for smaller than second increments in the normal case.

https://github.com/bndtools/bnd/blob/5d8500fd174eb7ec4228f383baba910d4d5573f8/biz.aQute.repository/src/aQute/bnd/repository/osgi/OSGiRepository.java#L152